Client/game_sa/CPoolSAInterface.h

```diff
@@ -149,3 +149,16 @@ struct SPoolData
         }
     }
 };
+
+template <class T, class I>
```

Comment:
`class I` is unused?

Reply:
Yeah, i've removed this. I didn't understand why this exists